### PR TITLE
Convert from sass to scss

### PR DESCRIPTION
**Pull request by madesch**
Solving #3

### CODE REVIEW

File: src/styles/_shame.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    }
+
+    &--incorrect {
+      color: color(red, dark);
+      text-decoration: line-through;
+
+      &::before {
+        content: '💩';
+      }
+    }
```
Comment (zsoltime) [inline src/styles/_shame.scss]: This block should be outside of the for loop. The for loop only contains the `nth-of-type` selector in the original: 

https://github.com/codesect/gulp-starter/blob/master/src/styles/_shame.sass#L42-L44

Comment (madesch) [reply src/styles/_shame.scss]: Oh right, my bad :)

**Comment by madesch** [comment]
There you go :)

**Comment by zsoltime** [comment]
Looks great. Thanks a lot for your help :)

**Pull request merged**